SalesforceFoundation/EDA
# Adding reentrancy checks to the AFFL_AccRecordType_TDTM class.

# Critical Changes

# Changes
* We've updated the AFFL_AccRecordType_TDTM class to prevent reentrancy issues.

# Issues Closed
* Fixes #532 

# New Metadata

# Deleted Metadata

# Testing Notes
1. Create an account with an Educational Institution record type.
2. Create a new contact. Specify the account you just created as the Primary Educational Institution (you may need to drag this field onto the contact page layout).
3. Verify that an affiliation is automatically creating linking the contact to the account.
4. Change the account record type to Business Organization.
5. Verify that the Primary Educational Institution on the contact is now blank. The Primary Business Organization is set to the account instead.
